packages/server/api/src/app/app-connection/app-connection-service/app-connection-service.ts

```diff
@@ -189,6 +196,81 @@ export const appConnectionService = (log: FastifyBaseLogger) => ({
         }))
     },
 
+    async applyOperation(params: ApplyOperationParams): Promise<PopulatedFlow[]> {
```

Comment:
Remove the Operation idea, we only need UPDATE_CONNECTION,  will be moved listFlowsFromAppConnection to flows,

I think this file is to large, you can start new file for this.

app-connection-replace-utils.ts, all it does list from flows service and replace it.